ui/src/app/components/apps-panel/utils.tsx

```diff
@@ -168,3 +174,35 @@ export const toUserEnvironmentStatusByAppType = (
 export const showAppsPanel = (config: ConfigResponse) => {
   return config.enableCromwellGKEApp || config.enableRStudioGKEApp;
 };
+
+export interface AppState {
+  appType: UIAppType;
+  active: boolean;
+}
+
+const getAppState = (
+  runtime: Runtime | null | undefined,
+  userApps: UserAppEnvironment[],
+  appType: UIAppType
+): AppState => {
+  return {
+    appType,
+    active:
+      appType === UIAppType.JUPYTER
+        ? isRuntimeActive(runtime?.status)
+        : isAppActive(findApp(userApps, appType)),
+  };
+};
+
+export const getAppsByDisplayGroup = (
```

Comment:
could you add a return type and/or comment for this?  I'm finding it a little difficult to picture how to consume the result of this function.

Reply:
Good call. I added a clarifying comment. If that is not clear, please let me know.

Reply:
Not seeing it - do you have an unpushed commit?

Reply:
Sorry, I posted this before I actually committed it. I saw @dmohs's comment this morning and held off committing it. I left the comment as a response to @dmohs's comment:

```
```Returns a two element array. The first element is an array of apps
that are in the active state. The second element is an array of apps
that are in the available state (not active). 
```